### DISCUSSION TITLE
Billing AWS Transfer Family when I configure Storage as AWS S3

### DISCUSSION THREAD

**Discussion by aarens66** [drakkan/sftpgo#1462] [General]
Hi all,
I have a question, I'm billed 0,30$/hour by AWS Transfer Family in order to use AWS S3 as backend storage. Is it normal?
If yes, do I have to mount the bucket in order to avoid this fee?

Thanks!

**Comment by drakkan** [comment]
Hello,

you don't need AWS Transfer Family, just configure your S3 bucket(s). SFTPGo can replace AWS Transfer Family if it fits your use case

**Reply by aarens66** [reply]
Hello,

I configured AWS S3 (compatible) at Filesystem->storage in the UI and it seems to enable AWS Transfer Family (Time each protocol is enabled). How can I configure to avoid this?

In my case I need to use a bucket s3 as a storage due to a storage limit

Thanks!

**Comment by drakkan** [comment]
> Hello,
> 
> I configured AWS S3 (compatible) at Filesystem->storage in the UI and it seems to enable AWS Transfer Family (Time each protocol is enabled). How can I configure to avoid this?
> 
> In my case I need to use a bucket s3 as a storage due to a storage limit
> 
> Thanks!

SFTPGo does not enable any AWS resources, it only accesses the configured bucket using the provided credentials

**Reply by aarens66** [reply]
Hello,

Okey, thank you for your reply!
